packages/liveblocks-react-comments/src/__tests__/index.test.tsx

```diff
@@ -124,10 +124,10 @@ const server = setupServer(
 
 beforeAll(() => server.listen());
 afterEach(() => {
-  MockWebSocket.reset();
+  MockWebSocket.instances = [];
 });
 beforeEach(() => {
-  MockWebSocket.reset();
```

Comment:
Was this accidental?

Reply:
No, in `beta` when I rebased on `main` I saw your change to this syntax and I used it everywhere but turns out we don't use the same MSW versions across packages and so we had to [revert it for the Redux and Zustand packages](https://github.com/liveblocks/liveblocks/pull/1437/commits/5c963a3bddf60d7d919548972eeca2718de38f5e). This package is also affected but we just didn't catch it because its tests aren't running in CI, so I'm just changing it back now.

Reply:
Oh, oof! Sorry I broke stuff for you unknowingly.